### PR TITLE
Fernet keys sync polish

### PR DESCRIPTION
**Pull request by skazi0**
Some followup changes for fernet-keys-sync feature. See commit messages for details.

### CODE REVIEW

**Comment by skazi0** [comment]
[Reinstalling node in existing cluster fails, because fernet-keys folder does not exist](https://trello.com/c/SBMOaFz1/187-reinstalling-node-in-existing-cluster-fails-because-fernet-keys-folder-does-not-exist)

**Comment by vuntz** [comment]
@skazi0 you can rebase now that the other patch is merged.

**Comment by skazi0** [comment]
@vuntz thx! rebased.

File: chef/cookbooks/keystone/recipes/ha.rb [hunk 1]
```diff
@@ -72,11 +72,28 @@
...
+  mode "0755"
+  action fernet_resources_action
+end
+
 template "/usr/bin/keystone-fernet-keys-sync.sh" do
   source "keystone-fernet-keys-sync.sh"
   owner "root"
   group "root"
   mode "0755"
+  action fernet_resources_action
```
Comment (ilausuch) [inline chef/cookbooks/keystone/recipes/ha.rb]: We always will take an action create or delete. Sometimes will create a file over existing one, or other ways delete an non existing one? Could it generates a problem? The same for the flowing actions.

Comment (skazi0) [reply chef/cookbooks/keystone/recipes/ha.rb]: Chef's meaning for those actions is more like "create" = "make sure it exists and matches the template" and "delete" = "make sure it doesn't exist". I don't expect any problems with these.

File: chef/cookbooks/keystone/recipes/server.rb [hunk 2]
```diff
@@ -391,13 +391,6 @@
   package "rsync"
   crowbar_pacemaker_sync_mark "sync-keystone_install_rsync" if ha_enabled
 
-  template "/usr/bin/keystone-fernet-keys-push.sh" do
-    source "keystone-fernet-keys-push.sh"
-    owner "root"
-    group "root"
-    mode "0755"
-  end
-
```
Comment (vuntz) [inline chef/cookbooks/keystone/recipes/server.rb]: Given that this script is ued a few line below, I don't know of moving it is really a good idea (unless you move the all thing).

Comment (skazi0) [reply chef/cookbooks/keystone/recipes/server.rb]: This script is used in two places. Directly in `rsync_command` below and inside `keystone-fernet-keys-sync.sh` alert handler. We could move the whole HA related code from `server.rb` to `ha.rb` but it is mixed with some non-HA code so some duplication would be needed.

**Pull request merged**